app/src/main/res/values/strings.xml

```diff
@@ -294,10 +294,10 @@
     <string name="acra_report_toast">Sorry, the application crashed. An anonymous bug report will be sent to the
         developers
     </string>
-    <string name="season">Season</string>
+    <string name="season">S</string>
     <string name="season_format">%1$s %2$d%3$s</string>
     <string name="no_season">No Season</string>
-    <string name="episode">Episode</string>
+    <string name="episode">E</string>
```

Comment:
This change would impact all other uses of this in the app (result UI and player UI), when changing things it is preferable to only change what you intend or need to change, otherwise it will introduce bugs or unwanted behavior. This change would also only affect English users and changed translations are less likely to be fixed that non-existing translations.

You are looking for episode_short and season_short, which already exists and is translated in other languages already.